extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -141,6 +141,15 @@ describe('amp-analytics', function() {
     return waitForSendRequest(analytics, 100);
   }
 
+  function clearIframePing(config) {
```

Comment:
Apologies, I missed the file separator. Yeah, we are very lax about jsdoc in tests.